src/main/java/org/gridsuite/voltageinit/server/service/VoltageInitResultContext.java

```diff
@@ -60,16 +60,16 @@ public static VoltageInitResultContext fromMessage(Message<String> message) {
 
     public Message<String> toMessage() {
         return MessageBuilder.withPayload("")
-                .setHeader("resultUuid", resultUuid.toString())
-                .setHeader("networkUuid", runContext.getNetworkUuid().toString())
-                .setHeader(PARAMETERS_UUID_HEADER, runContext.getParametersUuid() != null ? runContext.getParametersUuid().toString() : null)
-                .setHeader(VARIANT_ID_HEADER, runContext.getVariantId())
-                .setHeader(HEADER_RECEIVER, runContext.getReceiver())
-                .setHeader(HEADER_USER_ID, runContext.getUserId())
-                .setHeader(REPORT_UUID_HEADER, runContext.getReportInfos().reportUuid() != null ? runContext.getReportInfos().reportUuid().toString() : null)
-                .setHeader(REPORTER_ID_HEADER, runContext.getReportInfos().reporterId())
-                .setHeader(REPORT_TYPE_HEADER, runContext.getReportInfos().computationType())
-                .setHeader(VOLTAGE_LEVELS_IDS_RESTRICTED, runContext.getVoltageLevelsIdsRestricted())
+                .setHeader("resultUuid", getResultUuid().toString())
+                .setHeader("networkUuid", getRunContext().getNetworkUuid().toString())
+                .setHeader(PARAMETERS_UUID_HEADER, getRunContext().getParametersUuid() != null ? getRunContext().getParametersUuid().toString() : null)
```

Comment:
Need only override getSpecificMsgHeaders() for PARAMETERS_UUID_HEADER and VOLTAGE_LEVELS_IDS_RESTRICTED

Reply:
I forced the use of objectMapper everywhere in order to do this. Tell me what you think of it : https://github.com/gridsuite/voltage-init-server/pull/68/commits/ab64ffa03d56f777540ec99179a1a9ef8d9fa68c